### ISSUE TITLE
Nan Loss

### ISSUE THREAD

**Issue by sulei1998** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#290]
Hello, when i try to train, i find the loss is nan. I did not change any code. Could you give me some advise?
![image](https://user-images.githubusercontent.com/56378342/205558035-54219d66-af03-4f78-b854-c86259075600.png)

my cmd:
``` python
python train.py --train-images-folder /root/Source/sl_dataset/coco2017/train2017/ --prepared-train-labels prepared_train_annotation.pkl --val-labels val_subset.json --val-images-folder /root/Source/sl_dataset/coco2017/val2017/ --checkpoint-path checkpoints/mobilenet_sgd_68.848.pth.tar --from-mobilenet
```

**Comment by sulei1998** [comment]
And i do not know why the speed of loading the model is very slow. It needs about 30 mins to strat the train. My GPU is 3090 * 2.

**Comment by sulei1998** [comment]
I find the problem. My cuda version is wrong.
